src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/AbaccusReportITest.java

```diff
@@ -0,0 +1,146 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.time.Clock;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/letter/LetterController_initSummonsReminderLetter.sql"
+})
+@SuppressWarnings("PMD.LawOfDemeter")
+public class AbaccusReportITest extends AbstractStandardReportControllerITest {
+    @Autowired
+    public AbaccusReportITest(TestRestTemplate template) {
+        super(template, AbaccusReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getBureauJwt();
+    }
+
+    @Autowired
+    private Clock clock;
+
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .fromDate(LocalDate.of(2024, 1, 1))
+            .toDate(LocalDate.of(2024, 1, 31))
+            .build());
+    }
+
+    @Test
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+    void positiveTypicalBureau() {
+        testBuilder()
+            .jwt(getBureauJwt())
```

Comment:
Not needed (Will default to the valid JWT if one is not provided

Reply:
Removed